docker/local-develop.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+function dependency(){
+	(command -v $1 > /dev/null || (echo "install $1" && exit 1))
+}
+
+dependency jq     || exit 1
```

Comment:
this is the same as:
`(commanv -v jq >/dev/null || (echo "install jq" && exit 1)) || exit 1`
1. It has duplicates for exiting
2. It hard to read (look at the first mistake)

you basically can use `which` command and check those statements in the if statements, e.g (didn't test it):
```bash
if [ -x "$(which jq)" ]; then
    echo install jq
    exit 1
fi
```
or generify this block via your function

Reply:
function can not exit with `exit 1` as I understand, so I have to write like this.

You propose to inline function instead of `dependency` name?

Reply:
Yep, that's it
also remove common tools, e.g sed, grep